libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -180,7 +182,7 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
         noinput: true,
       };
       if (blockHeight) {
-        query.block_height = blockHeight;
+        query.from_height = blockHeight;
```

Comment:
Nope, no issue even with more than 2k txs. 
Example: `js:1:ethereum:0x57af10eD3469b2351AE60175d3C9B3740E1Bb649:` -> https://live.ledger.tools/sync 
=> operationsCount: `3713`

But that means we've basically been doing full synchro everytime instead of incremental. 

Even if the batchSize is 2k txs, the `fetchAllTransactions` is looping to get more than that through pagination.

Reply:
During dev I tested it with 10 txs  batch_size and it was working well, maybe we've been doing incremental synchro with default parameters

Can you confirm it still works well by setting manually the batch_size to something very low @lvndry ?

Reply:
I confirm that the `block_height` query parameter is ignored when doing the request and thus we've been doing full synchro all the time